code/modules/mob/living/silicon/robot/drone/drone_items.dm

```diff
@@ -219,7 +219,7 @@
 /obj/item/weapon/gripper/verb/drop_item_verb()
 	set name = "Drop Item"
 	set desc = "Release an item from your magnetic gripper."
-	set category = "Drone"
+	set category = statpanel("Status")
```

Comment:
мб вкладка Object? IC? что за коммандс...

Reply:
Честно, хз, если поискать в коде, предполагаю, что можно будет найти это перенаправление. Когда пытаешься пихнуть verb в  Status. Или типа того, в IC или Object я думаю будет не очень пихать, там своя сортировка и придется каждый раз искать нужную кнопку по названию, а тут и отдельная вкладка и кнопки на виду.

Reply:
> Перенес Drop item во вкладку Commands.

> set category = statpanel("Status")

Reply:
> > Перенес Drop item во вкладку Commands.
> 
> > set category = statpanel("Status")

?

Reply:
Это ты писал, все вопросы к тебе

Reply:
Если написать тупо - set category = "Status", то он создаст ещё одну вкладку Status и помесит туда кнопку.

Reply:
> > Перенес Drop item во вкладку Commands.
> 
> > set category = statpanel("Status")

Ну, вместо того, чтобы поместить verb в Status, оно создает вкладку Commands и помещает кнопку туда. 
Могу сейчас порыскать в коде, если это принципиально, на локалке у меня работает вышеописанным образом, хотя я менял там только пару файлов боргов.

Reply:
То есть это даже не ты придумал название категории?

Reply:
Да, я взял его из кода выше. 

/mob/living/silicon/robot/Stat()
	..()
>	if(statpanel("Status"))
		if(cell)
			stat(null, text("Charge Left: [round(cell.percent())]%"))
			stat(null, text("Cell Rating: [round(cell.maxcharge)]")) // Round just in case we somehow get crazy values
			stat(null, text("Power Cell Load: [round(used_power_this_tick)]W"))
		else
			stat(null, text("No Cell Inserted!"))
		if(module)
			var/obj/item/weapon/tank/jetpack/current_jetpack = locate(/obj/item/weapon/tank/jetpack) in module.modules
			if(current_jetpack) // if you have a jetpack, show the internal tank pressure
				stat("Internal Atmosphere Info: [current_jetpack.name]")
				stat("Tank Pressure: [current_jetpack.air_contents.return_pressure()]")
		if(locate(/obj/item/device/gps/cyborg))
			var/turf/T = get_turf(src)
			stat(null, "GPS: [COORD(T)]")

		stat(null, text("Lights: [lights_on ? "ON" : "OFF"]"))
		
		
Хотел проверить, что будет в таком случае. В таком случае, оно создает Commands.

Reply:
> statpanel("Status")

> "Commands"

100 процентное сходство

Reply:
> > statpanel("Status")
> 
> > "Commands"
> 
> 100 процентное сходство

Тк искать в коде? Или мне банально заменить statpanel("Status") на "Commands", чтобы это не вызывало вопросов?

Reply:
Сделать нормально, писать словами, а не давать бьонду самому выбирать

Reply:
> Сделать нормально, писать словами, а не давать бьонду самому выбирать

Готово